manager/src/main/java/com/redhat/service/bridge/manager/ProcessorService.java

```diff
@@ -41,16 +42,22 @@ public Processor getProcessor(String processorId, String bridgeId, String custom
 
     public Processor createProcessor(String bridgeId, String customerId, ProcessorRequest processorRequest) {
         Bridge bridge = getAvailableBridge(bridgeId, customerId);
-        Processor p = processorDAO.findByBridgeIdAndName(bridgeId, processorRequest.getName());
-        if (p != null) {
+        if (processorDAO.findByBridgeIdAndName(bridgeId, processorRequest.getName()) != null) {
             throw new AlreadyExistingItemException("Processor with name '" + processorRequest.getName() + "' already exists for bridge with id '" + bridgeId + "' for customer '" + customerId + "'");
         }
 
-        p = new Processor();
+        final Processor p = new Processor();
+
+        Set<Filter> filters = null;
+        if (processorRequest.getFilters() != null) {
+            filters = processorRequest.getFilters().stream().map(x -> new Filter(x.getKey(), x.getType(), x.getStringValue(), p)).collect(Collectors.toSet());
```

Comment:
Here we convert the `ProcessorRequest` to the `Process` entity, and we store the `value` as string